readme.md

```diff
@@ -2,109 +2,106 @@
 
-### Potential Project Ideas
+[See the requirements for the contents of the submission here.](evaluation.md)
 
-For this project, we want you to work with your team to build a creative product that you actually think someone will want to use. We won't have time to do tons of customer research, but take some time to brainstorm. If you're struggling for ideas, the ones below could help get you started.
+Feedback will be returned to you by **9:00 am, Monday, August 31st**.
 
-##### Bucketli.st
-Besides finishing WDI, you surely have one or two things you'd love to do with your life. Let's get 'em on paper! You could integrate with a third-party location-based API to allow users to search for a location or venue to add to their bucket list items.
+Note that instructors will *not* be grading on a numbered scale this time. Projects will simply be either "meets expectations" or "does not meet expectations".
+
+---
 
-##### Survey App
-Imagine sending out a survey to everyone in the class – what should we eat for lunch today? Or 1-5, how well did you understand what we just learned? It would be even more awesome if it were realtime, so you could see answers pouring in as they're submitted.
+### Attendance
 
-##### Hello, Comments
-Imagine the benefits of having an API where you could embed comments into any website you want. They could even update in realtime if you wanted, so that you'd never have to refresh the page. CMS providers across the world could quit writing code from scratch and just embed your widget instead.
+Attendance will be taken at 9:00am on Tuesday, and at 12:00pm on Wednesday. Groups are otherwise welcome to work outside of GA at their collective discretion.
 
 ---
 
-### Useful Resources
+### Support
 
-* **[Sequelize](http://docs.sequelizejs.com/en/latest/)**
-* **[Express JS](http://expressjs.com/)**
-* **[Getting Starts with Node.js on Heroku](https://devcenter.heroku.com/articles/getting-started-with-nodejs)**
+Instructors will be checking Github issues to respond to technical questions.
+
+Additionally, each group will receive 3 "tokens" for 20 minutes of in-person instructor support. These may be redeemed any time during business hours: simply Slack the instructor and they'll come as quickly as possible. Unused minutes don't "roll over".
+
+For existential questions, and/or those regarding expectations, instructors may be reached via Slack at any time, inside and outside of business hours (although they may be somewhat slower to respond outside of business hours).
 
 ---
 
-### Project Feedback + Evaluation
+### Presentations
 
-* __Project Workflow__: Did you complete the user stories, wireframes, task tracking, and/or ERDs, as specified above? Did you use source control as expected for the phase of the program you’re in (detailed above)?
+Each group will present their project to the class starting at **1:30pm on Friday, August 28th**.
```

Comment:
if each group has 15 minutes to present

1:30 - 230 - first 4
2:40 - 3:40 - second 4
3:50 - 4:50 - third 4